security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,27 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-5-0"]
+== OpenShift Compliance Operator 1.5.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.5.0:
+
+* link:https://access.redhat.com/errata/RHBA-2024:3533[RHBA-2024:3533 - OpenShift Compliance Operator 1.5.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-5-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* With this update, the Compliance Operator provides a unique profile ID for easier programmatic use.  (link:https://issues.redhat.com/browse/CMP-2450[*CMP-2450*])
+
+* With this release, the Compliance Operator is now tested and supported on the ROSA HCP environment. The Compliance Operator loads only Node profiles when running on ROSA HCP. This is because a Red Hat managed platform restricts access to the control plane, which makes Platform profiles irrelevant to the operator's function.(link:https://issues.redhat.com/browse/CMP-2581[*CMP-2581*])
```

Comment:
Nit - per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces) there should be a non breaking space in between Red and Hat:
```suggestion
* With this release, the Compliance Operator is now tested and supported on the ROSA HCP environment. The Compliance Operator loads only Node profiles when running on ROSA HCP. This is because a Red{nbsp}Hat managed platform restricts access to the control plane, which makes Platform profiles irrelevant to the operator's function.(link:https://issues.redhat.com/browse/CMP-2581[*CMP-2581*])
```